macros/sql/star.sql

```diff
@@ -17,8 +17,14 @@
       {{- return('*') -}}
     {%- else -%}
         {%- for col in cols %}
-            {%- if relation_alias %}{{ relation_alias }}.{% else %}{%- endif -%}{{ adapter.quote(col)|trim }} {%- if prefix!='' or suffix!='' %} as {{ adapter.quote(prefix ~ col ~ suffix)|trim }} {%- endif -%}
+            {%- if relation_alias %}{{ relation_alias }}.{% else %}{%- endif -%}
+                {%- if quote_identifiers -%}
+                    {{ adapter.quote(col)|trim }} {%- if prefix!='' or suffix!='' %} as {{ adapter.quote(prefix ~ col ~ suffix)|trim }} {%- endif -%}
+                {% elif not quote_identifiers %}
```

Comment:
Can this just be `else`?

I didn't notice this earlier when we were looking at this together in person 😅 

```suggestion
                {% else %}
```

Reply:
It could! I had it as that, but a coworker of yours suggested I be even more specific.
Shall I remove it?

Reply:
I just finished adding all the changes that we spoke of!

Reply:
Yep, I'd remove it. It will be more readable without it.

Reply:
Got it!